express/routes/players.js

```diff
@@ -0,0 +1,56 @@
+const express = require('express');
+const httpStatus = require('http-status');
+const { validate } = require('express-validation');
+
+const services = require('../services/players');
+const validations = require('../validations/players')
+
+const router = express.Router();
+
+router.get('/', async (req, res) => {
+	const players = await services.getAll();
+
+	return res.send({ players })
+});
+
+router.get('/:id', validate(validations.get), async (req, res) => {
+	const player = await services.getById(req.params.id);
+
+	if (!player) {
+		return res.sendStatus(httpStatus.NOT_FOUND);
+	}
+
+	return res.send({ player });
+});
+
+router.post('/', validate(validations.post), async (req, res) => {
+	const player = await services.create(req.body.player);
+
+	return res.status(httpStatus.OK).send({ player });
```

Comment:
No need to send ok here, and it should be CREATED as far as I understand

Reply:
I'd say there should be OK when the object is created and returned, CREATED when the object is created, but only its reference is returned (e.g. Id)